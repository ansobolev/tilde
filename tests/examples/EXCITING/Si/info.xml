<?xml version="1.0"?>
<?xml-stylesheet href="http://xml.exciting-code.org/info.xsl" type="text/xsl"?>
<info date="2013-09-27" time="14:53:47" versionhash="" title="Bulk Silicon indirect gap test">
  <groundstate status="finished">
    <scl>
      <iter iteration="1" rms="1.0000000000000000" rmslog10="0.00000000000" deltae="0.0000000000000000" deltaelog10="-Infinity" chgdst="5.2428847334937835E-002" chgdstlog10="-1.28042969029" fermidos="0.00000000000">
        <energies totalEnergy="-579.739493223" fermiEnergy="0.161671760134" sum-of-eigenvalues="-317.019819574" electronic-kinetic="577.669640334" core-electron-kinetic="566.489765714" Coulomb="-1117.68679313" Coulomb-potential="-842.210367340" nuclear-nuclear="-87.7475273348" electron-nuclear="-1217.66816426" Hartree="187.728898459" Madelung="-696.581609464" xc-potential="-52.4790925679" exchange="-37.5750372253" correlation="-2.14730319863"/>
        <charges totalcharge="28.00000000" core="20.00000000" core_leakage="0.4630667346E-02" valence="8.000000000" interstitial="3.803542959" muffin-tin-total="24.19645704">
          <atom species="Si" muffin-tin="12.09822852"/>
          <atom species="Si" muffin-tin="12.09822852"/>
        </charges>
        <timing timetot="2.28420257568" timeinit="1.39741110802" timemat="0.312874555588" timefv="0.241674184799" timesv="0.00000000000" timerho="0.168481826782" timepot="0.163760900497" timefor="0.00000000000"/>
      </iter>
      <iter iteration="2" rms="1.2815407120945543E-002" rmslog10="-1.89226759256" deltae="579.73949322342605" deltaelog10="2.76323288654" chgdst="1.2914785142999532E-002" chgdstlog10="-1.88891281458" fermidos="0.258483370962E-26">
        <energies totalEnergy="-577.353088234" fermiEnergy="0.205478178423" sum-of-eigenvalues="-315.361843805" electronic-kinetic="579.535233829" core-electron-kinetic="567.826059611" Coulomb="-1117.22525985" Coulomb-potential="-842.496256404" nuclear-nuclear="-87.7475273348" electron-nuclear="-1216.45920862" Hartree="186.981476109" Madelung="-695.977131646" xc-potential="-52.4008212293" exchange="-37.5185503838" correlation="-2.14451183085"/>
        <charges totalcharge="28.00000000" core="20.00000000" core_leakage="0.4661672770E-02" valence="8.000000000" interstitial="3.944521155" muffin-tin-total="24.05547885">
          <atom species="Si" muffin-tin="12.02773942"/>
          <atom species="Si" muffin-tin="12.02773942"/>
        </charges>
        <timing itertime="1.39764189720" timetot="3.18890357018" timeinit="1.39741110802" timemat="0.622300624847" timefv="0.480632066727" timesv="0.00000000000" timerho="0.344228744507" timepot="0.344331026077" timefor="0.00000000000"/>
      </iter>
      <iter iteration="3" rms="7.8937556489534714E-003" rmslog10="-2.10271632131" deltae="577.35308823430944" deltaelog10="2.76144149318" chgdst="1.3244502711349953E-003" chgdstlog10="-2.87796434343" fermidos="0.362975560151E-34">
        <energies totalEnergy="-577.633114985" fermiEnergy="0.203674050588" sum-of-eigenvalues="-315.536297976" electronic-kinetic="579.369134959" core-electron-kinetic="567.979970128" Coulomb="-1117.32950681" Coulomb-potential="-842.491802794" nuclear-nuclear="-87.7475273348" electron-nuclear="-1216.67215616" Hartree="187.090176685" Madelung="-696.083605417" xc-potential="-52.4136301411" exchange="-37.5278771926" correlation="-2.14486593748"/>
        <charges totalcharge="28.00000000" core="20.00000000" core_leakage="0.4636697162E-02" valence="8.000000000" interstitial="3.929569091" muffin-tin-total="24.07043091">
          <atom species="Si" muffin-tin="12.03521545"/>
          <atom species="Si" muffin-tin="12.03521545"/>
        </charges>
        <timing itertime="1.37028908730" timetot="4.08171200752" timeinit="1.39741110802" timemat="0.931447267532" timefv="0.720278263092" timesv="0.00000000000" timerho="0.500276565552" timepot="0.532298803329" timefor="0.00000000000"/>
      </iter>
      <iter iteration="4" rms="5.4528180004287678E-004" rmslog10="-3.26337899759" deltae="0.28002675052459836" deltaelog10="-0.552800479192" chgdst="2.5850557139903544E-003" chgdstlog10="-2.58753009241" fermidos="0.258039337251E-53">
        <energies totalEnergy="-578.081712979" fermiEnergy="0.199938146091" sum-of-eigenvalues="-315.817053412" electronic-kinetic="579.054145238" core-electron-kinetic="567.775672253" Coulomb="-1117.44912271" Coulomb-potential="-842.439079988" nuclear-nuclear="-87.7475273348" electron-nuclear="-1216.96411076" Hartree="187.262515387" Madelung="-696.229582716" xc-potential="-52.4321186624" exchange="-37.5412614536" correlation="-2.14547405372"/>
        <charges totalcharge="28.00000000" core="20.00000000" core_leakage="0.4606448623E-02" valence="8.000000000" interstitial="3.900043433" muffin-tin-total="24.09995657">
          <atom species="Si" muffin-tin="12.04997828"/>
          <atom species="Si" muffin-tin="12.04997828"/>
        </charges>
        <timing itertime="1.48020696640" timetot="4.96659851074" timeinit="1.39741110802" timemat="1.24284696579" timefv="0.961382150650" timesv="0.00000000000" timerho="0.652277469635" timepot="0.712680816650" timefor="0.00000000000"/>
      </iter>
      <iter iteration="5" rms="6.5693842672549845E-005" rmslog10="-4.18247533392" deltae="0.44859799395965183" deltaelog10="-0.348142672827" chgdst="2.4639378500131739E-004" chgdstlog10="-3.60837025095" fermidos="0.127648572023E-51">
        <energies totalEnergy="-578.079768615" fermiEnergy="0.200227822658" sum-of-eigenvalues="-315.815448201" electronic-kinetic="579.074384080" core-electron-kinetic="567.801216847" Coulomb="-1117.46689346" Coulomb-potential="-842.457011314" nuclear-nuclear="-87.7475273348" electron-nuclear="-1216.98172093" Hartree="187.262354807" Madelung="-696.238387798" xc-potential="-52.4328209668" exchange="-37.5418020875" correlation="-2.14545715235"/>
        <charges totalcharge="28.00000000" core="20.00000000" core_leakage="0.4603530989E-02" valence="8.000000000" interstitial="3.902222728" muffin-tin-total="24.09777727">
          <atom species="Si" muffin-tin="12.04888864"/>
          <atom species="Si" muffin-tin="12.04888864"/>
        </charges>
        <timing itertime="1.40290403366" timetot="5.88294291496" timeinit="1.39741110802" timemat="1.54844069481" timefv="1.19935989380" timesv="0.00000000000" timerho="0.808617591858" timepot="0.929113626480" timefor="0.00000000000"/>
      </iter>
      <iter iteration="6" rms="2.2983899755001789E-006" rmslog10="-5.63857628120" deltae="1.9443635079596788E-003" deltaelog10="-2.71122253841" chgdst="2.6079651645346271E-005" chgdstlog10="-4.58369821392" fermidos="0.855001570141E-52">
        <energies totalEnergy="-578.080482992" fermiEnergy="0.200199954778" sum-of-eigenvalues="-315.816013879" electronic-kinetic="579.071763380" core-electron-kinetic="567.798022463" Coulomb="-1117.46503291" Coulomb-potential="-842.455017855" nuclear-nuclear="-87.7475273348" electron-nuclear="-1216.97999330" Hartree="187.262487725" Madelung="-696.237523987" xc-potential="-52.4327594040" exchange="-37.5417541773" correlation="-2.14545927993"/>
        <charges totalcharge="28.00000000" core="20.00000000" core_leakage="0.4603686549E-02" valence="8.000000000" interstitial="3.901975959" muffin-tin-total="24.09802404">
          <atom species="Si" muffin-tin="12.04901202"/>
          <atom species="Si" muffin-tin="12.04901202"/>
        </charges>
        <timing itertime="1.44187188148" timetot="6.81282901764" timeinit="1.39741110802" timemat="1.86912441254" timefv="1.44235515594" timesv="0.00000000000" timerho="0.966950654984" timepot="1.13698768616" timefor="0.00000000000"/>
      </iter>
      <iter iteration="7" rms="2.2558561229976685E-007" rmslog10="-6.64668860282" deltae="7.1437667997997778E-004" deltaelog10="-3.14607273094" chgdst="4.1531694908246786E-007" chgdstlog10="-6.38162034497" fermidos="0.856579280261E-52">
        <energies totalEnergy="-578.080295534" fermiEnergy="0.200199964423" sum-of-eigenvalues="-315.815888076" electronic-kinetic="579.071872198" core-electron-kinetic="567.798092086" Coulomb="-1117.46496064" Coulomb-potential="-842.455009312" nuclear-nuclear="-87.7475273348" electron-nuclear="-1216.97985731" Hartree="187.262423997" Madelung="-696.237455988" xc-potential="-52.4327509619" exchange="-37.5417479817" correlation="-2.14545910652"/>
        <charges totalcharge="28.00000000" core="20.00000000" core_leakage="0.4603723469E-02" valence="8.000000000" interstitial="3.901980534" muffin-tin-total="24.09801947">
          <atom species="Si" muffin-tin="12.04900973"/>
          <atom species="Si" muffin-tin="12.04900973"/>
        </charges>
        <timing itertime="1.67330098152" timetot="7.71177387238" timeinit="1.39741110802" timemat="2.18055939674" timefv="1.68199181557" timesv="0.00000000000" timerho="1.12439370155" timepot="1.32741785049" timefor="0.00000000000"/>
      </iter>
      <iter iteration="8" rms="6.7505825522817501E-009" rmslog10="-8.17065874742" deltae="1.8745815020793088E-004" deltaelog10="-3.72709567280" chgdst="4.2572889311664695E-008" chgdstlog10="-7.37086687438" fermidos="0.856950603713E-52">
        <energies totalEnergy="-578.080306932" fermiEnergy="0.200199979106" sum-of-eigenvalues="-315.815895507" electronic-kinetic="579.071869818" core-electron-kinetic="567.798094007" Coulomb="-1117.46496914" Coulomb-potential="-842.455013668" nuclear-nuclear="-87.7475273348" electron-nuclear="-1216.97986994" Hartree="187.262428136" Madelung="-696.237462305" xc-potential="-52.4327516568" exchange="-37.5417484965" correlation="-2.14545911476"/>
        <charges totalcharge="28.00000000" core="20.00000000" core_leakage="0.4603721012E-02" valence="8.000000000" interstitial="3.901980655" muffin-tin-total="24.09801935">
          <atom species="Si" muffin-tin="12.04900967"/>
          <atom species="Si" muffin-tin="12.04900967"/>
        </charges>
        <timing itertime="2.26535606384" timetot="8.61109161377" timeinit="1.39741110802" timemat="2.48995113373" timefv="1.92155551910" timesv="0.00000000000" timerho="1.27734494209" timepot="1.52482891083" timefor="0.00000000000"/>
      </iter>
      <iter iteration="9" rms="5.3520735274524468E-010" rmslog10="-9.27147792881" deltae="1.1398400715734169E-005" deltaelog10="-4.94315607928" chgdst="2.9481227087228772E-009" chgdstlog10="-8.53045444394" fermidos="0.856917775243E-52">
        <energies totalEnergy="-578.080306825" fermiEnergy="0.200199977767" sum-of-eigenvalues="-315.815895449" electronic-kinetic="579.071869635" core-electron-kinetic="567.798093717" Coulomb="-1117.46496886" Coulomb-potential="-842.455013443" nuclear-nuclear="-87.7475273348" electron-nuclear="-1216.97986961" Hartree="187.262428082" Madelung="-696.237462139" xc-potential="-52.4327516405" exchange="-37.5417484842" correlation="-2.14545911481"/>
        <charges totalcharge="28.00000000" core="20.00000000" core_leakage="0.4603721064E-02" valence="8.000000000" interstitial="3.901980630" muffin-tin-total="24.09801937">
          <atom species="Si" muffin-tin="12.04900968"/>
          <atom species="Si" muffin-tin="12.04900968"/>
        </charges>
        <timing itertime="1.62031316757" timetot="9.49855065346" timeinit="1.39741110802" timemat="2.79631710052" timefv="2.15930843353" timesv="0.00000000000" timerho="1.43030905724" timepot="1.71520495415" timefor="0.00000000000"/>
      </iter>
      <structure>
        <crystal unitCellVolume="270.0113940" BrillouinZoneVolume="0.9186657265" nktot="0" ngridk="6    6    6">
          <basevect>5.130000000       5.130000000       0.000000000</basevect>
          <basevect>5.130000000       0.000000000       5.130000000</basevect>
          <basevect>0.000000000       5.130000000       5.130000000</basevect>
          <reciprvect>0.6123962288      0.6123962288     -0.6123962288</reciprvect>
          <reciprvect>0.6123962288     -0.6123962288      0.6123962288</reciprvect>
          <reciprvect>-0.6123962288      0.6123962288      0.6123962288</reciprvect>
        </crystal>
        <species chemicalSymbol="Si">
          <atom x="0.00000000000" y="0.00000000000" z="0.00000000000"/>
          <atom x="0.250000000000" y="0.250000000000" z="0.250000000000"/>
        </species>
      </structure>
    </scl>
  </groundstate>
</info>
