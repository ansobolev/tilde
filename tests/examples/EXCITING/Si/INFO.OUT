================================================================================
| EXCITING BERYLLIUM started                                                   =
| version hash id:                                                             =
|                                                                              =
| Date (DD-MM-YYYY) : 27-09-2013                                               =
| Time (hh:mm:ss)   : 14:53:47                                                 =
|                                                                              =
| All units are atomic (Hartree, Bohr, etc.)                                   =
================================================================================

********************************************************************************
| Ground-state run starting from atomic densities                              *
********************************************************************************

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| Starting initialization                                                      +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

 Lattice vectors (cartesian) :
      5.1300000000      5.1300000000      0.0000000000
      5.1300000000      0.0000000000      5.1300000000
      0.0000000000      5.1300000000      5.1300000000

 Reciprocal lattice vectors (cartesian) :
      0.6123962288      0.6123962288     -0.6123962288
      0.6123962288     -0.6123962288      0.6123962288
     -0.6123962288      0.6123962288      0.6123962288

 Unit cell volume                      :     270.0113940000
 Brillouin zone volume                 :       0.9186657265

 Species :    1 (Si)
     parameters loaded from            :    Si.xml
     name                              :    silicon
     nuclear charge                    :     -14.00000000
     electronic charge                 :      14.00000000
     atomic mass                       :   51196.73454000
     muffin-tin radius                 :       2.10000000
     # of radial points in muffin-tin  :     297

     atomic positions (lattice) :
       1 :   0.00000000  0.00000000  0.00000000
       2 :   0.25000000  0.25000000  0.25000000

 Total number of atoms per unit cell   :       2

 Spin treatment                        :    spin-unpolarised

 Number of Bravais lattice symmetries  :      48
 Number of crystal symmetries          :      48

 k-point grid                          :       6    6    6
 Total number of k-points              :      16
 k-point set is reduced with crystal symmetries

 R^MT_min * |G+k|_max (rgkmax)         :       7.00000000
 Species with R^MT_min                 :       1 (Si)
 Maximum |G+k| for APW functions       :       3.33333333
 Maximum |G| for potential and density :      14.00000000

 G-vector grid sizes                   :      36    36    36
 Total number of G-vectors             :   12387

 Maximum angular momentum used for
     APW functions                     :      10
     computing H and O matrix elements :       5
     potential and density             :       6
     inner part of muffin-tin          :       2

 Total nuclear charge                  :     -28.00000000
 Total electronic charge               :      28.00000000
 Total core charge                     :      20.00000000
 Total valence charge                  :       8.00000000

 Number of empty states                :      17
 Total number of valence states        :      22

 Total number of local-orbitals        :       8

 Exchange-correlation type :    3
     Perdew-Wang/Ceperley-Alder, Phys. Rev. B 45, 13244 (1992)

 Smearing scheme                       :    Gaussian
 Smearing width                        :       0.00100000

 Using multisecant Broyden potential mixing

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| Ending initialization                                                        +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++

********************************************************************************
| Groundstate module started                                                   *
********************************************************************************
 Output level for this task is set to normal

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| Self-consistent loop started                                                 +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Density and potential initialised from atomic data

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| SCF iteration number :    1                                                  +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Total energy                               :      -579.739493223426
 ___________________________________________________________________
 Fermi energy                               :         0.161671760134
 Kinetic energy                             :       577.669640334370
 Coulomb energy                             :     -1117.686793133901
 Exchange energy                            :       -37.575037225264
 Correlation energy                         :        -2.147303198632

 DOS at Fermi energy (states/Ha/cell)       :         0.000000000000

 Electron charges :
     core                                   :        20.00000000
     core leakage                           :         0.00463067
     valence                                :         8.00000000
     interstitial                           :         3.80354296
     charge in muffin-tin spheres :
                  atom     1    Si          :        12.09822852
                  atom     2    Si          :        12.09822852
     total charge in muffin-tins            :        24.19645704
     total charge                           :        28.00000000

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| SCF iteration number :    2                                                  +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Total energy                               :      -577.353088234309
 ___________________________________________________________________
 Fermi energy                               :         0.205478178423
 Kinetic energy                             :       579.535233828567
 Coulomb energy                             :     -1117.225259848213
 Exchange energy                            :       -37.518550383815
 Correlation energy                         :        -2.144511830848

 DOS at Fermi energy (states/Ha/cell)       :         0.000000000000

 Electron charges :
     core                                   :        20.00000000
     core leakage                           :         0.00466167
     valence                                :         8.00000000
     interstitial                           :         3.94452115
     charge in muffin-tin spheres :
                  atom     1    Si          :        12.02773942
                  atom     2    Si          :        12.02773942
     total charge in muffin-tins            :        24.05547885
     total charge                           :        28.00000000

 Wall time (seconds)                        :         4.11

 RMS change in effective potential (target) :  0.128154E-01  ( 0.100000E-05)
 Absolute change in total energy   (target) :   577.353      ( 0.100000E-03)
 Charge distance                   (target) :  0.129148E-01  ( 0.100000E-04)

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| SCF iteration number :    3                                                  +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Total energy                               :      -577.633114984834
 ___________________________________________________________________
 Fermi energy                               :         0.203674050588
 Kinetic energy                             :       579.369134958780
 Coulomb energy                             :     -1117.329506813578
 Exchange energy                            :       -37.527877192558
 Correlation energy                         :        -2.144865937478

 DOS at Fermi energy (states/Ha/cell)       :         0.000000000000

 Electron charges :
     core                                   :        20.00000000
     core leakage                           :         0.00463670
     valence                                :         8.00000000
     interstitial                           :         3.92956909
     charge in muffin-tin spheres :
                  atom     1    Si          :        12.03521545
                  atom     2    Si          :        12.03521545
     total charge in muffin-tins            :        24.07043091
     total charge                           :        28.00000000

 Wall time (seconds)                        :         5.45

 RMS change in effective potential (target) :  0.789376E-02  ( 0.100000E-05)
 Absolute change in total energy   (target) :  0.280027      ( 0.100000E-03)
 Charge distance                   (target) :  0.132445E-02  ( 0.100000E-04)

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| SCF iteration number :    4                                                  +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Total energy                               :      -578.081712978794
 ___________________________________________________________________
 Fermi energy                               :         0.199938146091
 Kinetic energy                             :       579.054145238352
 Coulomb energy                             :     -1117.449122709848
 Exchange energy                            :       -37.541261453576
 Correlation energy                         :        -2.145474053722

 DOS at Fermi energy (states/Ha/cell)       :         0.000000000000

 Electron charges :
     core                                   :        20.00000000
     core leakage                           :         0.00460645
     valence                                :         8.00000000
     interstitial                           :         3.90004343
     charge in muffin-tin spheres :
                  atom     1    Si          :        12.04997828
                  atom     2    Si          :        12.04997828
     total charge in muffin-tins            :        24.09995657
     total charge                           :        28.00000000

 Wall time (seconds)                        :         6.89

 RMS change in effective potential (target) :  0.545282E-03  ( 0.100000E-05)
 Absolute change in total energy   (target) :  0.448598      ( 0.100000E-03)
 Charge distance                   (target) :  0.258506E-02  ( 0.100000E-04)

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| SCF iteration number :    5                                                  +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Total energy                               :      -578.079768615286
 ___________________________________________________________________
 Fermi energy                               :         0.200227822658
 Kinetic energy                             :       579.074384079881
 Coulomb energy                             :     -1117.466893455347
 Exchange energy                            :       -37.541802087469
 Correlation energy                         :        -2.145457152350

 DOS at Fermi energy (states/Ha/cell)       :         0.000000000000

 Electron charges :
     core                                   :        20.00000000
     core leakage                           :         0.00460353
     valence                                :         8.00000000
     interstitial                           :         3.90222273
     charge in muffin-tin spheres :
                  atom     1    Si          :        12.04888864
                  atom     2    Si          :        12.04888864
     total charge in muffin-tins            :        24.09777727
     total charge                           :        28.00000000

 Wall time (seconds)                        :         8.26

 RMS change in effective potential (target) :  0.656938E-04  ( 0.100000E-05)
 Absolute change in total energy   (target) :  0.194436E-02  ( 0.100000E-03)
 Charge distance                   (target) :  0.246394E-03  ( 0.100000E-04)

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| SCF iteration number :    6                                                  +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Total energy                               :      -578.080482991966
 ___________________________________________________________________
 Fermi energy                               :         0.200199954778
 Kinetic energy                             :       579.071763379795
 Coulomb energy                             :     -1117.465032914527
 Exchange energy                            :       -37.541754177305
 Correlation energy                         :        -2.145459279929

 DOS at Fermi energy (states/Ha/cell)       :         0.000000000000

 Electron charges :
     core                                   :        20.00000000
     core leakage                           :         0.00460369
     valence                                :         8.00000000
     interstitial                           :         3.90197596
     charge in muffin-tin spheres :
                  atom     1    Si          :        12.04901202
                  atom     2    Si          :        12.04901202
     total charge in muffin-tins            :        24.09802404
     total charge                           :        28.00000000

 Wall time (seconds)                        :         9.66

 RMS change in effective potential (target) :  0.229839E-05  ( 0.100000E-05)
 Absolute change in total energy   (target) :  0.714377E-03  ( 0.100000E-03)
 Charge distance                   (target) :  0.260797E-04  ( 0.100000E-04)

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| SCF iteration number :    7                                                  +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Total energy                               :      -578.080295533816
 ___________________________________________________________________
 Fermi energy                               :         0.200199964423
 Kinetic energy                             :       579.071872198265
 Coulomb energy                             :     -1117.464960643896
 Exchange energy                            :       -37.541747981663
 Correlation energy                         :        -2.145459106522

 DOS at Fermi energy (states/Ha/cell)       :         0.000000000000

 Electron charges :
     core                                   :        20.00000000
     core leakage                           :         0.00460372
     valence                                :         8.00000000
     interstitial                           :         3.90198053
     charge in muffin-tin spheres :
                  atom     1    Si          :        12.04900973
                  atom     2    Si          :        12.04900973
     total charge in muffin-tins            :        24.09801947
     total charge                           :        28.00000000

 Wall time (seconds)                        :        11.30

 RMS change in effective potential (target) :  0.225586E-06  ( 0.100000E-05)
 Absolute change in total energy   (target) :  0.187458E-03  ( 0.100000E-03)
 Charge distance                   (target) :  0.415317E-06  ( 0.100000E-04)

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| SCF iteration number :    8                                                  +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Total energy                               :      -578.080306932216
 ___________________________________________________________________
 Fermi energy                               :         0.200199979106
 Kinetic energy                             :       579.071869818148
 Coulomb energy                             :     -1117.464969139141
 Exchange energy                            :       -37.541748496467
 Correlation energy                         :        -2.145459114756

 DOS at Fermi energy (states/Ha/cell)       :         0.000000000000

 Electron charges :
     core                                   :        20.00000000
     core leakage                           :         0.00460372
     valence                                :         8.00000000
     interstitial                           :         3.90198065
     charge in muffin-tin spheres :
                  atom     1    Si          :        12.04900967
                  atom     2    Si          :        12.04900967
     total charge in muffin-tins            :        24.09801935
     total charge                           :        28.00000000

 Wall time (seconds)                        :        13.56

 RMS change in effective potential (target) :  0.675058E-08  ( 0.100000E-05)
 Absolute change in total energy   (target) :  0.113984E-04  ( 0.100000E-03)
 Charge distance                   (target) :  0.425729E-07  ( 0.100000E-04)

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| Convergence targets achieved. Performing final SCF iteration                 +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 Total energy                               :      -578.080306824682
 ___________________________________________________________________
 Fermi energy                               :         0.200199977767
 Kinetic energy                             :       579.071869634789
 Coulomb energy                             :     -1117.464968860475
 Exchange energy                            :       -37.541748484185
 Correlation energy                         :        -2.145459114812

 DOS at Fermi energy (states/Ha/cell)       :         0.000000000000

 Electron charges :
     core                                   :        20.00000000
     core leakage                           :         0.00460372
     valence                                :         8.00000000
     interstitial                           :         3.90198063
     charge in muffin-tin spheres :
                  atom     1    Si          :        12.04900968
                  atom     2    Si          :        12.04900968
     total charge in muffin-tins            :        24.09801937
     total charge                           :        28.00000000

++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
| Self-consistent loop stopped                                                 +
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 STATE.OUT is written

********************************************************************************
| Groundstate module stopped                                                   *
********************************************************************************
 Total time spent (seconds)            :        15.56

================================================================================
| EXCITING BERYLLIUM stopped                                                   =
================================================================================
